guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -56,6 +56,11 @@ include::topics/upgrading_discovery_satellite.adoc[leveloffset=+3]
 // Configuring Subnets with a Template Capsule
 include::topics/upgrading_discovery_subnet_with_a_template_capsule.adoc[leveloffset=+3]
 
+ifdef::katello,satellite[]
```

Comment:
this applies to foreman on EL too, so should be `ifdef::foreman-el,katello,satellite` I think